physics/module_sf_ruclsm.F90

```diff
@@ -2049,6 +2068,9 @@ SUBROUTINE SFCTMP (debug_print, delt,ktau,conflx,i,j,         & !--- input varia
           eeta = eetas*(one-snowfrac) + eeta*snowfrac
           qfx = qfxs*(one-snowfrac) + qfx*snowfrac
           hfx = hfxs*(one-snowfrac) + hfx*snowfrac
+          !IF ( add_fire_heat_flux ) then ! JLS
```

Comment:
Any reason to keep this commented-out section here and in line 2119?

Reply:
@grantfirl The commented out section is removed.